### ISSUE TITLE
Test Output results

### ISSUE THREAD

**Issue by TroySchmidt** [rtbenfield/vscode-jest-test-adapter#48]
Where can we see the test results output.  For example, using the vscode-jest extension it adds the failure line with red squiggles and puts the expected versus received values on the line that assertion failed on.  Is that done in this test adapter or in the explorer UI?

**Comment by mikeborozdin** [comment]
Agreed. The extension looks like, but it doesn't show what exactly is failing in a test.

**Comment by Chris2011** [comment]
Exactly, that feature is a must have IMHO. :) Great job anyway @rtbenfield
